packages/ui-react/src/containers/AccountModal/forms/CancelSubscription.tsx

```diff
@@ -48,17 +48,25 @@ const CancelSubscription = () => {
     navigate(modalURLFromLocation(location, null), { replace: true });
   };
 
+  const onFinishUnsubscription = () => {
+    if (location.state?.returnToPathname) {
+      navigate(location.state.returnToPathname, { replace: true });
+    } else {
+      closeHandler();
+    }
+  };
+
   if (!subscription) return null;
 
   return (
-    <React.Fragment>
+    <div style={{ padding: '16px' }}>
```

Comment:
How come this is needed?

Reply:
Not exactly needed, just a remark from a designer for the sake of better visuals.